### ISSUE TITLE
Support for CakePHP 3.x

### ISSUE THREAD

**Issue by fabaff** [destinydriven/cakephp-highcharts-plugin#26]
Do you plan to update the plugin for CakePHP 3.x?

**Comment by destinydriven** [comment]
Yes, I do actually plan to update to 3.x but I haven't gotten around to it as it.   It's on my to do list. As always, any help along the way is always welcome.

**Comment by destinydriven** [comment]
@fabaff  I started work on a cake 3.* branch but it's still rough and a bit lacking on documentation. But it should be enough to get you going if you really need 3.0 support. I hope to move it to github once it's in a decent enough state.  You can find it here for now: https://bitbucket.org/destinydriven/cakephp3-highcharts

**Comment by eidercarlos** [comment]
Hello, how is the plugin for CakePHP3? I can't install using composer. Do you know if it is currently working?

**Comment by aleDVM** [comment]
I want to know what if this plugin work with cakephp3

**Comment by destinydriven** [comment]
Hey guys,

I started work on a cake 3.* version some time ago but it's not quite
completed.  You can take a stab at it if you like.

https://bitbucket.org/destinydriven/cakephp3-highcharts

**Comment by aleDVM** [comment]
[InvalidArgumentException]                                                   
  Could not find package destinydriven/cakephp3-highcharts at any version for  
   your minimum-stability (stable). Check the package spelling or your minimu  
  m-stability

**Comment by destinydriven** [comment]
Oh right, you likely won't get through with composer.  Might need to
submodule.   I will try to set up for composer later.

**Comment by aleDVM** [comment]
thanks ;P
